### PR TITLE
Add RSS generation

### PR DESCRIPTION
**Pull request by jonathanbardo**
@mcaskill I think we are MVP complete (or very close).
Right now this plugin adds a new endpoint (which looks like: /chimplet/monthly/?tax[category]=6,5) and output the RSS feed for the specified schedule and specified category.

Again, the last thing we need is to assign a user to a list group. Either the plugin or the theme could do that pretty easily using `chimplet/user/groupings` filter.

Note: the part to start the campaign automatically is commented out for now see [here](https://github.com/locomotivemtl/wordpress-chimplet/blob/develop/src/MailChimp/Facade.php#L419)

### CODE REVIEW

File: assets/rss/mailchimp-feed-rss2.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<?php
```
Comment (jonathanbardo) [inline assets/rss/mailchimp-feed-rss2.php:1]: @mcaskill I'll let you tweak this RSS template. Right now it should output the excerpt and the featured image of a post.

File: src/Chimplet/RSS.php [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+				'field'    => 'term_id',
+				'terms'    => $tax_ids,
+			];
+		}
+
+		// Build date_query
+		switch ( $schedule ) {
+			case 'monthly':
+				$date_query = [
+					[
```
Comment (jonathanbardo) [inline src/Chimplet/RSS.php:106]: @mcaskill The date_query might not meet business requirements. Feel free to adapt it to meet expectations.

Comment (mcaskill) [inline src/Chimplet/RSS.php:106]: @jonathanbardo The date_query might not be necessary. This just needs to be a normal functioning RSS Feed (like any of the default ones), just limited to the users interests. MailChimp will intelligently handle if the posts in the feed fit the campaign's schedule.

Based on the current options value for `posts_per_rss` (e.g., 10) each automated campaign will include no more than 10 posts. Maybe this count could be increased depending on Vogogo's expected output per chosen schedule.

Comment (jonathanbardo) [inline src/Chimplet/RSS.php:106]: I wasn't 100% sure that MailChimp would handle the date difference. I basically implemented that way so we can give MailChimp a more precise feed that would include all posts for a given period (I've actually filter the posts_per_page result to -1).

The default feed of WordPress doesn't really give a good overview of everything that happened in a certain time lapse in my opinion.

**Comment by mcaskill** [comment]
I've pushed some changes to the [`wwwvogogo/plugin-chimplet`](/vogogo/wwwvogogo/tree/plugin-chimplet) branch and to `chimplet/develop`.

Features:
- Email Type (_html_ or _text_) is now supported (re-enabled the UI on the WAVE Profile).
- MC Language is now set based on the WordPress locale.
- Updated the commit pointer for the Chimplet module in wwwvogogo.

Do we need to set `update_existing` for MailChimp's [`batchSubscribe`](https://apidocs.mailchimp.com/api/2.0/lists/batch-subscribe.php) when we sync changes ? Documentation indicates that we should.

**Pull request merged**